### ISSUE TITLE
Feature request: Multiple selections in queues and filelists

### ISSUE THREAD

**Issue by KipourosV** [airdcpp-web/airdcpp-webclient#90]
Also it would be nice if Remove downloaded files check box to be available on multiple selections too

**Comment by Nischi85** [comment]
How many upvotes would be needed? ;-)

**Comment by psychoocysp** [comment]
> How many upvotes would be needed? ;-)

I'm curious about this also. The feature is much needed.

**Comment by gunnarrt** [comment]
+1 for Multiple selections

**Comment by potloodzkie** [comment]
this is the number 1 needed feature indeed, its the most annoying thing to have to rightclick > download for every single directory you want to download. i expected this to have way more upvotes also

**Comment by gunnarrt** [comment]
+1 multiple selections

**Comment by kiam001** [comment]
yes, it would be cool if you could also mark only a few files in the filelist for downloading in a folder with 1024 files
